CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   #79, #83)
 
 ### Changed
+- Print authenticator version on startup, add version.go file for authenticator 
+  pkg, and remove VERSION references so that there is one place for versioning (#93)
```

Comment:
1. the printed message should be under `Added`.
2. I don't think we need to add here the change of the version file. It's not really customer facing.

Reply:
Also - this should go under `## [Unreleased]`, not `## [0.16.1]`

Reply:
dropped this commit

Reply:
i think we still need a changelog entry for this under "Added". just saying that the client now prints its version upon startup.